### PR TITLE
Correct typo

### PR DESCRIPTION
**Pull request by garethmcc**
## What did you implement:
Corrected a typo in the documentation

Closes #XXXXX

## Todos:

- [ ] Write tests
- [ ] Write documentation
- [ ] Fix linting errors
- [ ] Make sure code coverage hasn't dropped
- [ ] Provide verification config / commands / resources
- [ ] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: docs/providers/aws/events/apigateway.md [hunk 1]
```diff
@@ -12,45 +12,46 @@ layout: Doc
...
-  - [Resource Policy](#resource-policy)
-  - [Compression](#compression)
-  - [Binary Media Types](#binary-media-types)
-  - [AWS X-Ray Tracing](#aws-x-ray-tracing)
-  - [Tags / Stack Tags](#tags--stack-tags)
-  - [Logs](#logs)
+- [API Gateway](#API-Gateway)
+  - [Lambda Proxy Integration](#Lambda-Proxy-Integration)
+    - [Simple HTTP Endpoint](#Simple-HTTP-Endpoint)
+    - [Example "LAMBDA-PROXY" event (default)](#Example-%22LAMBDA-PROXY%22-event-default)
```
Comment (medikoo) [inline docs/providers/aws/events/apigateway.md]: I just checked, and this breaks link on GitHub, while currently it works on both GitHub on Website https://serverless.com/framework/docs/providers/aws/events/apigateway/#example-lambda-proxy-event-default

Are those link changes intended?

**Comment by garethmcc** [comment]
Those link changes were not intended. Not sure why it did that after merging master. I only edited one character on one line.

**Comment by garethmcc** [comment]
Can we cancel the PR I'll do it again

**Comment by medikoo** [comment]
> Can we cancel the PR I'll do it again

Yes, no problem!

**Comment by pmuens** [comment]
> I only edited one character on one line.

I think that's the answer. So it should be GTM :shipit:

**Pull request merged**